jboss-fuse-sap-tool-suite/site/pom.xml

```diff
@@ -12,5 +12,8 @@
 	<artifactId>org.fusesource.ide.sap.updatesite</artifactId>
 	<packaging>eclipse-repository</packaging>
 	<name>JBoss Tools Fuse Extras :: SAP Tooling :: Update Site</name>
+	<properties>
+		<skipDeployToJBossOrg>false</skipDeployToJBossOrg>

```

Comment:
Turns out we don't -- I just need to run the deployment from the root folder (instead of the site folder) and pass in `--projects site` so that maven can RESOLVE the locally built artifacts, but only DEPLOY the single project.